src/bundle/Resources/public/scss/ui/modules/muti-file-upload/_upload.item.scss

```diff
@@ -1,59 +1,86 @@
 .c-upload-list-item {
     display: flex;
     background: $ibexa-white;
-    padding: calculateRem(8px) calculateRem(32px);
+    padding: calculateRem(8px) 0;
+    min-height: calculateRem(60px);
 
-    &:first-child {
-        margin-top: calculateRem(16px);
+    &--errored {
+        background: $ibexa-color-danger-100;
+        color: $ibexa-color-danger;
+        border-radius: $ibexa-border-radius;
+
+        .ibexa-icon {
+            fill: $ibexa-color-danger;
+        }
+
+        .c-upload-list-item__size {
+            color: $ibexa-color-danger;
+        }
     }
 
     &__icon-wrapper {
         flex: 0 0 calculateRem(20px);
         display: flex;
+        justify-content: center;
+        align-items: center;
     }
 
     &__meta {
         padding: 0 calculateRem(16px);
         line-height: 1.4;
         max-width: 25vw;
+        display: flex;
+        justify-content: center;
+        align-items: center;
     }
 
     &__name {
         font-size: calculateRem(16px);
-        width: 100%;
-        text-overflow: ellipsis;
+        margin-right: calculateRem(8px);
+        max-width: 20vh;
         white-space: nowrap;
         overflow: hidden;
+        text-overflow: ellipsis;
     }
 
     &__size {
         color: $ibexa-color-base-dark;
-        font-size: calculateRem(14px);
+        font-size: $ibexa-text-font-size-medium;
     }
 
     &__info {
         flex: 1 1 auto;
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
     }
 
     &__message {
         font-style: italic;
-        font-size: calculateRem(12px);
+        font-size: $ibexa-text-font-size-small;
 
         &--error {
-            padding-top: calculateRem(3px);
-            color: $ibexa-color-warning-dark;
-            font-weight: bold;
+            .ibexa-icon {
+                margin-right: calculateRem(4px);
```

Comment:
Are there situations when we don't want this margin for icon? As I see both error and success has it